src/news-feed-events/NewsFeedEvent.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from "react";
+
+/**
+ * Represents an action taken.
+ */
+export default interface NewsFeedEvent {
+  "UNIQUE ID #": string;
+  "STATE": string;
+  "COUNTY": string;
+  "ACTION DATE": string;
+  // This column doesn't exist in the fake data.
+  "SOURCE"?: string;
+  "OVERALL BUCKET": string;
+  "SUB-CATEGORY": string;
+  "ACTION TAKEN": string;
+  "# OF PEOPLE RELEASED": string;
+  "# OF PEOPLE PREVENTED FROM ENTERING": string;
+  "SUMMARY RE-WRITE": string;
+}
+
+/**
+ * Convert a TSV spreadsheet into NewsFeedEvent objects.
+ */
+export function parseNewsFeedEventsFromTsv(tsv: string) {
+  let table = tsv.split("\n").map((line) => line.split("\t"));
+
+  const Headers = table[0] as (keyof NewsFeedEvent)[];
+  return table.slice(1).map((row) => {
+    let record = {} as NewsFeedEvent;
+    for (let i = 0; i < Headers.length; i++) {
+      let header = Headers[i];
+      let cell = row[i];
+      record[header] = cell;
+      1;
+    }
+    return record;
+  });
+}
+
+// Fake events
+
+const FakeNewsFeedEventTsv = `UNIQUE ID #	STATE	COUNTY	ACTION DATE	OVERALL BUCKET	SUB-CATEGORY	ACTION TAKEN	# OF PEOPLE RELEASED	# OF PEOPLE PREVENTED FROM ENTERING	SUMMARY RE-WRITE
+1	CA	<Null>	3/24/20	Curtail physical contact	Minimize facility transfers	Freeze prison intake for new sentences	0	0	CDCR stopped prison intake
+3	IN	<Null>	3/27/20	Curtail physical contact	Limit exposure between inmates/supervisees and others	Suspends visitation	0	0	All prison visits were suspended`;
+
+export const FakeNewsFeedEvents = parseNewsFeedEventsFromTsv(
+  FakeNewsFeedEventTsv,
+);
+
+// Real events
+
+// https://mnaoumov.wordpress.com/2018/02/05/download-google-drive-file-via-ajax-ignoring-cors/
+const NewsFeedEventsTsvUrl =
+  "https://cors-anywhere.herokuapp.com/https://docs.google.com/spreadsheets/d/e/2PACX-1vRccQtwTk0FiM3PxtBF5kF-O0RcdzOWyxZgolquqnNmReAEV7bmMSJE7PL_MT8HiQAEynhUJZrC0i6T/pub?gid=251892683&single=true&output=tsv";
```

Comment:
Nit: especially if we are going to have more than one data source, but even if we only have one, can we extract their URLs out to a shared location somewhere so they can be shared with confidence and we won't have to worry about minor typos to and fro?

Reply:
I agree! I don't know how it should be organized though, because I don't have a high-level overview of the data part of this project. Let me know what make the most sense. Alternatively, we can defer making this change until we have our second URL that we need, in which case the correct abstraction will become more obvious. What do you think?

Reply:
Sounds good!